invenio_app_ils/records/loaders/schemas/json/items.py

```diff
@@ -23,6 +23,18 @@ class Meta:
     value = fields.Str(required=True)
 
 
+class PriceSchema(Schema):
```

Comment:
`PriceSchema` is defined for `borrowing_requests` and for `order` too. Since all prices match that schema, maybe we can reuse the same marshmallow class definition for all.

Reply:
I will address it my ILL extension PR